Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -666,27 +675,26 @@ public void AddUnresolvedMemberDeclaration(UnboundMemberDeclaration declaration)
             }
         }
 
-        public void ClearStateCache(string projectId, bool notifyStateChanged = false)
+        public void ClearStateCache(string projectId)
         {
             try
             {
                 foreach (var moduleState in _moduleStates.Where(moduleState => moduleState.Key.ProjectId == projectId))
                 {
                     var qualifiedModuleName = moduleState.Key;
-                    var component = ProjectsProvider.Component(moduleState.Key);
-                    if (component != null)
+                    if (qualifiedModuleName.ComponentType == ComponentType.Undefined && qualifiedModuleName.ComponentType == ComponentType.ComComponent)
```

Comment:
Shouldn't that be an `||`?

Reply:
Good catch! I inverted the condition but forgot to invert the boolean operator.